cf/forest-snapshot-listing/.editorconfig

```diff
@@ -0,0 +1,13 @@
+# http://editorconfig.org
```

Comment:
What is this? Is this enforced in CI in any way?

Reply:
It was auto-created. Not sure who reads it.

Reply:
Deleted the file.